tools/nnyeah/nnyeah/NStrings.Designer.cs

```diff
@@ -0,0 +1,116 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace nnyeah {
+    using System;
+    using System.Reflection;
+    
+    
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
+    [System.Diagnostics.DebuggerNonUserCodeAttribute()]
+    [System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
+    internal class NStrings {
+        
+        private static System.Resources.ResourceManager resourceMan;
```

Comment:
Minor - `private` is the default visibility for members, so this is redundant.

Reply:
Good catch! The NStrings.Designer.cs is auto generated though so I am not sure if that will be changeable